docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -283,3 +283,54 @@ GET /transactions/{tx_id}
 
    :statuscode 200: A transaction with that ID was found.
    :statuscode 404: A transaction with that ID was not found.
+
+
+GET /unspents/
+-------------------------
+
+.. http:get:: /unspents?owner_after={owner_after}
+
+   Get a list of links to transactions' conditions that have not been used in
+   a previous transaction and could hence be called unspent conditions/outputs
+   (or simply: unspents).
+
+   This endpoint doesn't return anything if the querystring ``owner_after``
+   happens to not be defined in the request.
```

Comment:
There should be some response to tell the requester (user agent) that they did something wrong, i.e. their request was badly formed. The most general catch-all would be `HTTP 400 Bad Request`, but a more specific HTTP 40? might work too. Some discussion:

https://stackoverflow.com/questions/9454811/which-http-status-code-to-use-for-required-parameters-not-provided

https://stackoverflow.com/questions/3050518/what-http-status-response-code-should-i-use-if-the-request-is-missing-a-required